radiant/eclass.cpp

```diff
@@ -382,8 +382,8 @@ void Eclass_Init(){
 					if ( !strcmp( ValueForKey( g_qeglobals.d_project_entity, "gamemode" ), "sp" ) ) {
 						// SP mapping, ignore mp_*.def
 						char *name = (char *)pFile->data;
-						if ( name[0] == 'm' && name[1] == 'p' && name[2] == '_' 
-							|| name[0] == 'h' && name[1] == 'm' && name[2] == '_' ) {
+						if ( ( name[0] == 'm' && name[1] == 'p' && name[2] == '_' )
```

Comment:
Does that change anything? `a && b || c` already means `(a && b) || c` in the C grammar.